ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1349,6 +1349,8 @@ periodics:
       - --config=/etc/ci-secret-bootstrap/_config.yaml
       - --dry-run=false
       - --force=true
+      - --validate-bitwarden-items-usage
+      - --bw-allow-unused=openshift-ci-robot,openshift-merge-robot,openshift-build-robot,openshift-publish-robot,openshift-bugzilla-robot,openshift-cherrypick-robot,openshift-private-viewer,ci.dev.openshift.redhat.com,ci.openshift.redhat.com,jira-credentials-dptp-bot
```

Comment:
The flag doesn't work like that, you can not specify multiple values comma-seperated. You need to add the flag once per value you want to set.

/hold

Reply:
me: facepalm 

Let me fix this. Thanks for noticing.

Reply:
updated.